docs/prod-install.md

```diff
@@ -1,100 +1,190 @@
-Installing Zulip in production
-==============================
+# Install
```

Comment:
I think this should still mention "production" since we have something similar for dev.